packages/create-turbo/src/index.ts

```diff
@@ -29,7 +28,7 @@ const help = `
 
   If <dir> is not provided up front you will be prompted for it.
 
-  Flags:    
+  Flags:
     --use-npm           Explicitly tell the CLI to bootstrap the app using npm
```

Comment:
Can we only show this prompt if `--custom-registry` flag is passed into `create-turbo`?

Reply:
It's pointless, because event if one doesn't specific the flag, not registry value will get from the `npm config get registry` command. In this case, the installation should run properly in the default way, which is  using the default `registry.npmjs.org`

Reply:
Ahh I see. Cool.